build.cmd

```diff
@@ -1,108 +1,220 @@
 @echo off
-goto :main
+@if defined _echo echo on
 
 :main
+setlocal enabledelayedexpansion
```

Comment:
@brianrob I just added error checkins and fail fast (e.g. stop running subsequent steps if a project did not build, remove redundancy, and easier to reuse from other scripts such as buildapi)

---
In reply to: [100207724](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100207724) [](ancestors = 100207724)